src/Items/Board/Board.tsx

```diff
@@ -115,27 +118,29 @@ export function Board() {
   };
 
   return (
-    <DragDropContext onDragEnd={onDragEnd}>
-      <Droppable droppableId="all-columns" direction="horizontal" type="column">
-        {(provided) => (
-          <div
-            {...provided.droppableProps}
-            ref={provided.innerRef}
-            // isDraggingOver={snapshot.isDraggingOver}
-          >
-            <Stack direction="row">
-              {state.columnOrder.map((columnId, index) => {
-                const column = state.columns[columnId];
-
-                return (
-                  <Column key={columnId} column={column} tasks={column.taskIds} index={index} />
-                );
-              })}
-              {provided.placeholder}
-            </Stack>
-          </div>
-        )}
-      </Droppable>
-    </DragDropContext>
+    <Box sx={{ m: 3 }}>
```

Comment:
what is the purpose of the outer `Box`?

Reply:
only to add margin, maybe to be removed :thinking: